samples/bluetooth/mesh/light_switch/src/model_handler.c

```diff
@@ -40,9 +41,11 @@ static struct button buttons[] = {
 #if DT_NODE_EXISTS(DT_ALIAS(sw2))
 	{ .client = BT_MESH_ONOFF_CLI_INIT(&status_handler) },
 #endif
+#ifndef CONFIG_BT_MESH_LOW_POWER
 #if DT_NODE_EXISTS(DT_ALIAS(sw3))
```

Comment:
Can be combined into one `#if` using `defined`:
```suggestion
#if DT_NODE_EXISTS(DT_ALIAS(sw3)) && !defined(CONFIG_BT_MESH_LOW_POWER)
```